ambry-tools/src/integration-test/java/com.github.ambry/clustermap/HelixBootstrapUpgradeToolTest.java

```diff
@@ -40,6 +43,7 @@
 import static com.github.ambry.clustermap.HelixBootstrapUpgradeTool.*;
 import static com.github.ambry.clustermap.TestUtils.*;
 import static com.github.ambry.utils.TestUtils.*;
+import static org.hamcrest.CoreMatchers.*;
```

Comment:
Do we have to import this?

Reply:
It helps with one of the asserts. `hamcrest` comes with `junit`, so we are not bringing in anything new.